cbmc/proofs/IotMqtt_UnsubscribeAsync/cbmc-viewer.json

```diff
@@ -0,0 +1,23 @@
+{ "expected-missing-functions":
+  [
+    "IotLog_Generic",
+    "IotMqtt_Wait",
+    "IotMutex_Create",
+    "IotMutex_Destroy",
+    "IotMutex_Lock",
+    "IotMutex_Unlock",
+    "IotSemaphore_Create",
+    "IotSemaphore_Destroy",
+    "IotSemaphore_Post",
+    "IotSemaphore_TimedWait",
+    "IotSemaphore_Wait",
+    "IotTaskPool_CreateJob",
+    "IotTaskPool_GetSystemTaskPool",
+    "IotTaskPool_ScheduleDeferred",
+    "IotTaskPool_strerror",
+    "IotTaskPool_TryCancel",
+    "IotClock_GetTimeMs"
```

Comment:
Update this list with the one generated by the report. For instance, I believe you're not abstracting away `IotMqtt_Wait`.